tests/Fixtures/e2e/Unconfigured/run_tests.bash

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
the name of `Unconfigured` folder conflicts with the neighbor PR: https://github.com/infection/infection/pull/285/files#diff-c50be878dbd21bc70fc3c4da7deedccfR90

As I understand, it should have "_" to work properly

Reply:
It's not going to conflict, at least that's what `git merge` says to me. But I'm planning to rebase #285 over master once this merges anyway. Nothing to be concerned here.

Reply:
I mean not a git conflict, but logic conflict `Finder::create()->name('*_*')`. As far as I understand it searches all directories with `_` inside, thus `Unconfigured` will be skipped.

Anyway, let's merge it and adress (if it is a case) it in #285

Reply:
Good point, I will drop name requirement.

(Since this is a negative test, Infection fails with an error on it, #285 addresses this test [with a separate test case](https://github.com/infection/infection/pull/285/files#diff-c50be878dbd21bc70fc3c4da7deedccfR66) just for it. During normal E2E tests Infection is not expected to fail.)